yt/yt/server/scheduler/operations_cleaner.cpp

```diff
@@ -835,6 +839,10 @@ class TOperationsCleaner::TImpl
         result.TaskNames = attributes.FindYson("task_names");
         result.ControllerFeatures = attributes.FindYson("controller_features");
 
+        if (auto temporaryTokenNodeId = attributes.Find<TNodeId>("temporary_token_node_id")) {
+            result.DependentNodeIds = std::vector{*temporaryTokenNodeId};
```

Comment:
afaik, just `result.DependentNodeIds = {*temporaryTokenNodeId};` will be more conventional